doc/source/whatsnew/v0.19.0.rst

```diff
@@ -707,6 +707,7 @@ A ``Series`` will now correctly promote its dtype for assignment with incompat v
 
 
 .. ipython:: python
+   :okwarning:
```

Comment:
It feels weird silencing warnings in old whatsnew's, but I didn't see a better way

Reply:
Yeah, this is unfortunately the best we have right now.

Reply:
alt you can change these to code-blocks